packages/@okta/vuepress-site/docs/reference/api/asa/audits/index.md

```diff
@@ -47,11 +47,11 @@ This endpoint requires one of the following roles: `access_user`, `access_admin`
 | `prev`   |  boolean | (Optional) The direction of paging |
 
 
-#### Request body
+##### Request body
 
 This endpoint has no request body.
```

Comment:
Not the point of your ticket, but do we need to state "no request body" or is it more clear to just remove "Request Body" section completely?

Reply:
@albertchen-okta Good question. My opinion is that it makes for a cleaner page to remove them. I would think most developers know that if those sections aren't in the docs that they aren't required. But I defer to Chris and others :)